nn_surrogate_benchmark/model_evaluator.py

```diff
@@ -67,7 +67,9 @@ def evaluate_dataset(
         dataset_name: str,
     ) -> pd.DataFrame:
         X, y_true_scaled = self._extract_from_dataloader(dataloader)
+        y_true_scaled = y_true_scaled.flatten()
```

Comment:
why?

Reply:
Before that I was getting dimension mismatch errors. It seemed related to how the loader structured the batch data